Nhl.Api/README.md

```
@@ -1052,6 +1086,24 @@ Returns all the NHL player game center statistics for a specific player for a sp
 | gameType | [Nhl.Api.Enumerations.Game.GameType](#T-Nhl-Api-Enumerations-Game-GameType 'Nhl.Api.Enumerations.Game.GameType') | The game type for the NHL season for the player statistics |
 | cancellationToken | [System.Threading.CancellationToken](http://msdn.microsoft.com/query/dev14.query?appId=Dev14IDEF1&l=EN-US&k=k:System.Threading.CancellationToken 'System.Threading.CancellationToken') | A cancellation token to cancel the asynchronous operation |
 
+<a name='M-Nhl-Api-NhlApi-GetPlayoffBracketAsync-System-Int32,System-Threading-CancellationToken-'></a>
+### GetPlayoffBracketAsync(seasonYear,cancellationToken) `method`
+
+##### Summary
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)